client/src/modules/stock/depot-selection.modal.html

```diff
@@ -0,0 +1,72 @@
+<div class="modal-header" data-depot-selection-modal>
+  <ol class="headercrumb">
+    <li class="static" translate>TREE.DEPOTS</li>
+    <li class="title" translate>DEPOT.MODAL.SELECTION</li>
+  </ol>
+</div>
+
+<div class="modal-body">
+
+  <!-- link to depots page if depots not found -->
+  <p class="text-info">
+    <i class="fa fa-question-circle-o"></i>
+    <span translate>DEPOT.MODAL.NO_DEPOTS</span>
+
+    <a href ui-sref="depots.create">
+      <i class="fa fa-link"></i> <span translate>DEPOT.ADD_DEPOT</span>
+    </a>
+  </p>
+
+  <div uib-alert class="alert-warning" ng-if="$ctrl.hasNoDefaultDepot">
+    <p>
+      <strong> <i class="fa fa-warning"></i> <span translate>DEPOT.MODAL.NO_DEFAULT_DEPOT</span></strong>
+      <br />
+      <span translate>DEPOT.MODAL.SELECTION_EXPLANATION</span>
+      <a href ui-sref="depots.create"><b><span translate>DEPOT.ADD_DEPOT</span></b></a>.
+    </p>
+  </div>
+
+
+  <label class="control-label" translate>DEPOT.MODAL.SELECT_A_DEPOT</label>
+
+  <!-- list of depots -->
+  <ul class="list-group">
```

Comment:
What will happen if we have an institution with 10 depots ? the modal will still fit well in this situation? why not  to use select? (I like this user interface, I am just thinking in a laud voice).

Reply:
In theory, there shouldn't be _too_ many depots.  But with more depots, it looks like this:

![selectdepotwithatonofdepots](https://user-images.githubusercontent.com/896472/30484577-b34019ca-9a22-11e7-9b07-ef1de12383f5.png)
_Fig 1: Select depot with 14 depots_